mcs/class/Mono.Posix/Mono.Unix.Native/Stdlib.cs

```diff
@@ -378,7 +380,11 @@ static XPrintfFunctions ()
 	//
 	public class Stdlib
 	{
+#if NETSTANDARD2_0
+		internal const string LIBC = "c";
```

Comment:
That is correct.  I asked @migueldeicaza if we are to handle Windows at this time.  He said no.